metadata-service/auth-impl/src/main/java/com/datahub/authentication/token/TokenType.java

```diff
@@ -4,12 +4,23 @@
  * Represents a type of JWT access token granted by the {@link StatelessTokenService}.
  */
 public enum TokenType {
+
   /**
    * A UI-initiated session token
    */
-  SESSION,
+  SESSION(true),
   /**
    * A personal token for programmatic use
    */
-  PERSONAL,
+  PERSONAL(false);
+
+  private final boolean expirationMandatory;
+
+  private TokenType(boolean expirationMandatory) {
```

Comment:
Do we really need this argument "expirationMandatory"?

I don't like that this adds new complexity / states into our code when it feels like this doesn't need to have a special case.

Consider just always generating SESSION tokens with an expiration date. Personal tokens can begin to use a nullable argument to avoid setting an expiration.

Reply:
TLDR: Let's remove the expiration mandatory flag here and just trust the caller of "generateAccessToken" to do the right thing